packages/thirdweb/package.json

```diff
@@ -40,6 +40,11 @@
       "import": "./dist/esm/exports/chains.js",
       "default": "./dist/cjs/exports/chains.js"
     },
+    "./universal": {
```

Comment:
do we need this export since you have it in the root?

also, you renamed the file to bridge.ts - not universal

Reply:
Actually going to include it so users can import from root or do `thirdweb/bridge` since that's consistent with the other modules